### DISCUSSION TITLE
podman start container fails in SysVinit init.d script

### DISCUSSION THREAD

**Discussion by ojde** [containers/podman#22860] [Q&A]
I've a systemd-less Devuan Daedalus with podman 4.3.1 from the repo.

An /etc/init.d/startcontainers.sh script invokes "podman start container" and "podman stop container" respectively.

The (root-full) containers start (and stop) just fine when the init.d/startcontainers.sh script is called manually (being logged in as root).

On shutdown and boot, the script is executed as it should, but both "podman stop container" and "podman start container" return error code 1 and no containers are started in the latter case.

There are no entries in dmesg, syslog, messages, faillog, etc. shedding light on what goes wrong. I redirected all output from podman to a log, but there's no output. The logs of the containers are empty.

I tried to invoke the script from rc.local, it is invoked allright, same failure.
I tried to invoke the script from crontab -e, @reboot, it is invoked allright, same failure.
I tried with "run --rm -d", same failure.

What am I missing or doing wrong?

**Comment by PiotrBzdrega** [comment]
@ojde did you drop privileges for rootless command ??
Could you share your script?
I see that SysVinit runs all initialization scripts as root user.
You could drop root using ```/sbin/runuser username -s /bin/bash -c "podman start container"```
